### PR TITLE
finished testing with jest and enzyme

### PR DESCRIPTION
**Pull request by developing-stevie**
looking for input!

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -31,7 +33,8 @@ class Header extends Component{
                     SEE THE CODE
                   </NavLink>
                 </NavItem>
-                <UncontrolledDropdown inNavbar nav>
+                {/* adds dropdown menu */}
+                {/* <UncontrolledDropdown inNavbar nav>
```
Comment (WalkerTexas-coder) [inline src/components/Header.js:37]: I really don't love keeping this code for later. It's easy enough to grab from reactstrap let's take it out.

File: src/components/Header.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import Header from './Header'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()})
+
+
+describe("When Header renders", () => {
+    it("displays a NavbarBrand", () => {
+      const header = shallow(<Header />)
+      const headerHeading = header.find("h1").text()
```
Comment (WalkerTexas-coder) [inline src/components/Header.test.js:20]: IF we are goona put the .text() on the variable, let's rename the variable to reflect that. "headerHeadingtext"

File: src/components/Header.js [hunk 3]
```diff
@@ -7,17 +7,19 @@ import {
...
   NavLink,
-  UncontrolledDropdown,
-  DropdownToggle,
-  DropdownMenu,
-  DropdownItem,
+  // add drop down
+  // UncontrolledDropdown,
+  // DropdownToggle,
+  // DropdownMenu,
+  // DropdownItem,
```
Comment (WalkerTexas-coder) [inline src/components/Header.js:14]: Yea, I don't like this.

**Review by WalkerTexas-coder — APPROVED** [review]
Couple cosmetic changes but I am approving so you can merge as soon as you make them. Good work!

File: src/components/Header.js (hunk 3 above)
Comment (developing-stevie) [reply src/components/Header.js:14]: we will take this out

File: src/components/Header.js (hunk 1 above)
Comment (developing-stevie) [reply src/components/Header.js:37]: will do

File: src/components/Header.test.js (hunk 2 above)
Comment (developing-stevie) [reply src/components/Header.test.js:20]: we will do this on next commit

**Pull request merged**